### PR TITLE
Add Qwen pipeline and example

### PR DESCRIPTION
**Pull request by hkvision**

### CODE REVIEW

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+logger = logging.get_logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Predict Tokens using `generate()` API for npu model"
+    )
+    parser.add_argument(
+        "--repo-id-or-model-path",
+        type=str,
+        default="Qwen/Qwen2-7B-Instruct",
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py:34]: If "Qwen/Qwen2.5-7B-Instruct" can directly run with current code, maybe use "Qwen/Qwen2.5-7B-Instruct" as default ?
We can also update this in next PR 😊

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        help="The huggingface repo id for the Baichuan2 model to be downloaded"
+        ", or the path to the huggingface checkpoint folder",
+    )
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py:37]: Maybe add ` parser.add_argument("--lowbit-path", type=str, ...` here.

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    parser.add_argument('--prompt', type=str, default="AI是什么?",
+                        help='Prompt to infer')
+    parser.add_argument("--n-predict", type=int, default=32, help="Max tokens to predict")
+    parser.add_argument("--max-context-len", type=int, default=1024)
+    parser.add_argument("--max-prompt-len", type=int, default=960)
+    parser.add_argument("--disable-transpose-value-cache", action="store_true", default=False)
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py:43]: It looks like Qwen2 GW is already ready ? If so, maybe add  `parser.add_argument("--quantization_group_size", type=int, default=0)` here ?

**Review by rnwang04 — APPROVED** [review]
others LGTM.

**Review by plusbang — APPROVED** [review]
Please also update [readme](https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/README.md), other LGTM.

**Review by cyita — APPROVED** [review]

**Comment by hkvision** [comment]
> Please also update [readme](https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/README.md), other LGTM.

README updated.

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py (hunk 3 above)
Comment (hkvision) [reply python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen2.py]: Not tested yet, will test and update in a following PR.

**Comment by hkvision** [comment]
Merge it first. Will fix any issues in a following PR.

**Pull request merged**